package.json

```diff
@@ -45,6 +45,7 @@
     "eslint-plugin-prettier": "^3.0.0",
     "flow-bin": "^0.88.0",
     "jest": "^23.6.0",
+    "jest-expo": "^32.0.0",
```

Comment:
should it not be in `/apps/mobile/package.json`?

Reply:
Well we use it in `.jest.json` at the root of the repo, so I thought it'd be more appropriate there.